compiler/ilgen/VirtualMachineOperandStack.cpp

```diff
@@ -87,8 +87,25 @@ VirtualMachineOperandStack::Commit(TR::IlBuilder *b)
       }
    }
 
-void
-VirtualMachineOperandStack::MergeInto(OMR::VirtualMachineOperandStack *other, TR::IlBuilder *b)
+   void
+   VirtualMachineOperandStack::Reload(TR::IlBuilder* b)
+   {
+       TR::IlType* Element = _elementType;
+       TR::IlType* pElement = _mb->typeDictionary()->PointerTo(Element);
+       // reload the elements back into the simulated operand stack
+       // If the # of stack element has changed, the user should adjust the # of elements 
+       // using Drop to add/delete stack elements.
+       TR::IlValue* stack = b->Load("OperandStack_base");
+       for (int32_t i = _stackTop; i >= 0; i--) {
+           _stack[i] = b->LoadAt(Element,
+               b->IndexAt(pElement,
```

Comment:
not the indentation I normally apply here, though I suppose it's not mandatory. I like to line up the "b"s and insert indentation between the -> and the first operation to reflect the structure of the expression tree.